src/custom/components/OrdersPanel/index.tsx

```diff
@@ -28,10 +28,9 @@ const SideBar = styled.div`
   cursor: default;
   overflow-y: auto; // fallback for 'overlay'
   overflow-y: overlay;
```

Comment:
Not part of this PR, but this property appears twice. Which should we keep?

Reply:
This is to have a graceful fallback as this 'overlay' property isn't consistently applied: https://caniuse.com/css-overflow-overlay

In particular Safari.